src/tables.rs

```diff
@@ -1,10 +1,11 @@
 // DO NOT EDIT THIS FILE. IT WAS AUTOMATICALLY GENERATED BY:
 //
-//   yeslogic-ucd-generate canonical-combining-class --rust-enum ../../ucd-13.0.0
+//   yeslogic-ucd-generate canonical-combining-class --rust-enum /home/wmoore/Downloads/ucd-14.0
```

Comment:
Is this meant to include your local path? 🤔

Reply:
Ahh, seems to be  how it is done in https://github.com/yeslogic/unicode-general-category/pull/6, don't mind me!

Reply:
It doesn't really matter as long as it indicates the ucd version used.